src/MagnumPlugins/WavAudioImporter/Test/WavImporterTest.cpp

```diff
@@ -278,6 +312,10 @@ void WavImporterTest::mono32f() {
 
     CORRADE_COMPARE(importer.format(), Buffer::Format::MonoFloat);
     CORRADE_COMPARE(importer.frequency(), 48000);
+
+    CORRADE_COMPARE_AS(importer.data().slice(0, 16),
+        Containers::Array<char>::from(0, 0, 0, 0, 108, 57, -103, 59, 3, 63, 42, 60, -33, -81, -120, 60).slice(0, 16),
+        TestSuite::Compare::Container);
```

Comment:
@mosra I don't really think this would help with endian issues, given floating point endianness does not need to be the same as the rest of the systems endianness!